examples/s2n-mtls/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "s2n-mtls"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+s2n-quic = { version = "1", path = "../../quic/s2n-quic" }
+tokio = { version = "1", features = ["full"] }
```

Comment:
From https://github.com/aws/s2n-quic/blob/main/examples/rustls-mtls/Cargo.toml#L16-L17 can you add the following. It makes sure that we dont get transitive dependencies from local path and that the example can be built in isolation correctly.

```
[workspace]
members = ["."]
```

Reply:
OK, I’ll add